buildconfig/Jenkins/conda-setup

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# This script expects a environment variable called $WORKSPACE to be set.
```

Comment:
Can we avoid writing more scripts that depend on environment variables such as `WORKSPACE`? Can we pass these in as arguments?
I actually can't quite see where `WORKSPACE` is used?

Reply:
Left over from a previous iteration, necessary if we are going to put the mambaforge/miniconda in the WORKSPACE_DIR but yes we can just pass it from the configured bash script instead.